File: begin/emma.py


def emma(s):
    arr = s.split(' ')
    count = 0

    for word in arr:
        if word == 'Emma':
            count += 1

    return "Emma appeared {} times".format(count)


text1 = 'Emma is a good developer. Emma is a writer'
text2 = 'This does not say the word'

print(emma(text1))
print(emma(text2))

File: begin/emma.js


const emma = str => {
    let count = 0;
    let arr = str.split(' ');

    for (let x in arr) {
        if (arr[x] == 'Emma') {
            count++;
        }
    }
    return `Emma appeared ${count} times`;
}


let text1 = 'Emma is a good developer. Emma is a writer';
let text2 = 'This does not contain the word';

console.log(emma(text1));
console.log(emma(text2));